content/en/tracing/trace_collection/library_injection_local.md

```diff
@@ -103,7 +103,7 @@ The available library versions are listed in each container registry, as well as
 - [.NET][19]
   - **Note**: For .NET library injection, if the application container uses a musl-based Linux distribution (such as Alpine), you must specify a tag with the the `-musl` suffix for the pod annotation. For example, to use library version `v2.29.0`, specify container tag `v2.29.0-musl`.
 - [Ruby][20]
-  - **Note**: For Ruby, instrumenting only Ruby on Rails or Hanami applications is supported.
+  - **Note**: For Ruby, library injection support is in Beta. Instrumentation is only supported for Ruby on Rails or Hanami applications at this time.
 
 **Note**: If you already have an application instrumented using version X of the library, and then use library injection to instrument using version Y of the same tracer library, the tracer does not break. Rather, the library version loaded first is used. Because library injection happens at the admission controller level prior to runtime, it takes precedent over manually configured libraries.
```

Comment:
Is it possible to move all of this higher up at the top of the doc in the requirements section: https://docs.datadoghq.com/tracing/trace_collection/library_injection_local/?tab=kubernetes#requirements so you don't scroll through to the middle of the page to find out it's not compatible?